arangod/Pregel/GraphStore/GraphLoader.cpp

```diff
@@ -205,8 +205,8 @@ auto GraphLoader<V, E>::loadVertices(LoadableVertexShard loadableVertexShard)
   }
 
   std::string documentId;  // temp buffer for _id of vertex
-  auto cb = IndexIterator::makeDocumentCallbackF([&](LocalDocumentId token,
-                                                     VPackSlice slice) {
+  auto cb = [&](LocalDocumentId token, aql::DocumentData&& data,
```

Comment:
`data` is not used inside the function.
```suggestion
  auto cb = [&](LocalDocumentId token, aql::DocumentData&&,
```